libs/admin/src/lib/components/workgroups/workgroups.component.scss

```diff
@@ -0,0 +1,31 @@
+@use "../../styles/variables";
+
+.workgroups {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+
+  .workgroups__header {
```

Comment:
Remove the `workgroups__` prefix on all classes in this component as every element in this component's template is wrapped inside the `.workgroups` div anyways.